### PR TITLE
MNT: disable tree namespace until issues are resolved

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

Apply hotfix from dev/deployed hutch-python, add a comment for context.
This is the (dangerous) namespace that lets you do:
`xpp.ccm`, for example, instead of `xpp_ccm`. It creates these big cascading namespaces of devices.

## Motivation and Context

Tree namespace can cause havoc and break top-level devices
See #224, #159 

## How Has This Been Tested?

No complaints from its omission (yet) in dev/deployed hutch-python

### CODE REVIEW

**Review by klauer — APPROVED** [review]
@ZryletTC and I got bit by this recently, and it sure was a confusing one to debug.

👍 for disabling it for now - let's see if there are any complaints and then go from there.

**Comment by ZLLentz** [comment]
Ah I bet this fails a test that checks for the tree

File: hutch_python/tests/test_load_conf.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def test_file_load():
     err = '{} was overriden by a namespace'
     for elem in should_have:
         assert not isinstance(objs[elem], SimpleNamespace), err.format(elem)
-    assert 'tst' in objs
+    assert 'tst' not in objs # Tree namespace should be disabled
```
Comment (ZryletTC) [inline hutch_python/tests/test_load_conf.py:26]:
```suggestion
    assert 'tst' not in objs  # Tree namespace should be disabled
```

**Review by ZryletTC — APPROVED** [review]
👍 on disabling for now

Imports need to be commented out or noqa'd to appease the flake8 god:
```
hutch_python/load_conf.py:28:1: F401 '.namespace.tree_namespace' imported but unused
hutch_python/load_conf.py:31:1: F401 '.utils.count_ns_leaves' imported but unused
```

**Comment by ZLLentz** [comment]
There's something wrong with the travis configuration, we're installing hutch-python=v1.1.1 from pcds-tag instead of installing the version we built in the earlier build stage. I'm thinking there's some weird edge case where v1.1.1 has lighter dependencies and the ci helpers isn't using any sort of channel priority.

**Comment by klauer** [comment]
Didn't notice that before ☹️

Maybe if we add the following here:
https://github.com/pcdshub/pcds-ci-helpers/blob/d1bb15ace06cfd8fdda3f5ccad0981fcc59dfbe0/travis/shared_configs/anaconda-build.yml#L17
```
- if [ ! -z "$CONDA_BUILD_CHANNEL" ]; then conda config --add channels $CONDA_BUILD_CHANNEL; fi
```

and per-repo:
```
- CONDA_BUILD_CHANNEL="file://$TRAVIS_BUILD_DIR/bld-dir"
```

I think the TRAVIS_BUILD_DIR will still be correct when using the ci scripts

**Comment by ZLLentz** [comment]
Passes all tests locally and has a clean flake8
I think I merge?

**Comment by klauer** [comment]
Yeah, no reason to hold up this PR on the CI issue. Will open an issue in ci-helpers.

**Comment by ZLLentz** [comment]
The channel does still get added, you see it in the conda config output right before creating the env. But in this case, a random old build wins the install war.

**Comment by ZLLentz** [comment]
https://github.com/pcdshub/pcds-ci-helpers/blob/d1bb15ace06cfd8fdda3f5ccad0981fcc59dfbe0/travis/shared_configs/python-tester-conda.yml#L21

**Pull request merged**